.github/workflows/ci.yml

```diff
@@ -227,6 +227,99 @@ jobs:
+    needs: [ detect-changes ]
+    name: "[IT] Elasticsearch"
+    timeout-minutes: 25
+    permissions: {}  # GITHUB_TOKEN unused in this job
+    runs-on: gcp-perf-core-8-default
+    env:
+      ZEEBE_TEST_DOCKER_IMAGE: localhost:5000/camunda/zeebe:current-test
+    services:
+      # Set up an elastic search service container
+      # https://docs.github.com/en/actions/use-cases-and-examples/using-containerized-services/about-service-containers
+      elasticsearch:
+        image: docker.elastic.co/elasticsearch/elasticsearch:8.17.0
+        # Exposing the ports allow to be accessible via http://localhost:9200 in the runner
+        ports:
+          - 9200:9200
+        env:
+          # Disabling the security to make sure we can access without tls in our tests
+          # https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html#general-security-settings
+          xpack.security.enabled: "false"
+          # We run a single node for our integration tests
+          discovery.type: "single-node"
+          # Disabling to make sure we can delete indices via wildcards
+          # https://www.elastic.co/guide/en/elasticsearch/reference/current/index-management-settings.html
+          action.destructive_requires_name: "false"
+      # Set up a registry service container to host our custom docker image
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-build
+        with:
+          maven-cache-key-modifier: it-elasticsearch
+          vault-address: ${{ secrets.VAULT_ADDR }}
+          vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+          vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+      # Build the platform before running the integration tests; skipping frontend
+      - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
+        with:
+          maven-extra-args: -T1C -PskipFrontendBuild
+      - name: Create build output log file
+        run: echo "BUILD_OUTPUT_FILE_PATH=$(mktemp)" >> $GITHUB_ENV
+      # To fail early if set up with service container doesn't work and ES is not accessible
+      - name: Validate connection with ES
+        run: curl http://localhost:9200
+      # Run specific integration tests with ES service container
+      # During the migration of tests we run only specific package, where classes moved into
+      #
+      # To communicate to the IT what database to expect and run against:
+      # -Dcamunda.it.database.type=es
+      #
+      - name: Run integration test with externalized ES
+        run: >
+          ./mvnw -B -T 2 --no-snapshot-updates
+          -D forkCount=7
+          -D maven.javadoc.skip=true
+          -D skipUTs -D skipChecks
+          -D failsafe.rerunFailingTestsCount=3 -D flaky.test.reportDir=failsafe-reports
+          -P parallel-tests,extract-flaky-tests,skipFrontendBuild
+          -Dit.test=io/camunda/v2/**
```

Comment:
:question: What does this do? Selecting the right files? If that's the case, then :x: let's use junit tagging instead, that way of selecting files which are included is more flexible, and not hidden in the CI job but instead visible directly in the code.

Reply:
That is an intermediate state. Yes i wanted to run here only test that are migrated. After we migrated all i wanted to remove this again.

Reply:
I can add tags thanks 👍

Can we make sure that the tests are not executed by the other IT job? I guess `-DexcludedGroups=externalDb`

Reply:
I tried it with adding a [tag](https://github.com/camunda/camunda/pull/26852/commits/9e5d36cd9f7f479ad719beeca1c1eb64583d4c16) and running [maven with groups flag](https://github.com/camunda/camunda/pull/26852/commits/5b6334e14a3bdd50bc7c81735ef01bc8a9ecbfce), but it doesn't seem to work. It executed other tests as well

https://github.com/camunda/camunda/actions/runs/12886542433/job/35927356462#step:10:1

If you don't have a direct solution/hint I will go ahead and merge it first and find out this separately.

Reply:
Got it working with a profile. The issue seem to be that groups have been overwritten, before in build step

Reply:
https://github.com/camunda/camunda/pull/26852/commits/77de90be131f13209a6a0e88338c2f067e624a4d